website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,79 @@
 ---
 title: "Job notifications"
 id: "job-notifications"
-description: "Set up notifications in dbt Cloud to receive Email or Slack alerts for job run status."
+description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
+Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
 
-Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+## Email notifications
 
-### Email
+You can receive email alerts by configuring the dbt Cloud notification settings. As a **user**, you can set up email notifications for yourself. As an **admin**, you can set up notifications for yourself and other team members. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for details on license types that are eligible for email notifications.
```

Comment:
are **user** and **admin** the actual role names in dbt Cloud that should be used on this page? 

i continued to use those words as it was already there on the page but it didn't seem to match up when i looked at the pag https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users

Reply:
I think if we wanted to be more explicit we could say something like tihs:
- As a *developer user*, you can setup email notifications for yourself
- As an *account admin*, you can setup email notifications for yourself and other team members on your account
- As an *it or read only user*, you can _recieve_ email notifications, but not set them up

Reply:
thank you!